airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_bank_accounts.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "object": {
+      "type": ["string", "null"]
+    },
+    "account_holder_name": {
+      "type": ["string", "null"]
+    },
+    "account_holder_type": {
+      "type": ["string", "null"]
+    },
+    "account_type": {
+        "type": ["string", "null"]
+    },
+    "bank_name": {
+      "type": ["string", "null"]
+    },
+    "country": {
+      "type": ["string", "null"]
+    },
+    "currency": {
+      "type": ["string", "null"]
+    },
+    "fingerprint": {
+      "type": ["string", "null"]
+    },
+    "last4": {
+      "type": ["string", "null"]
+    },
+    "metadata": {
+      "type": "object"
```

Comment:
Do we need to add "null" here ? Because I see on stream `external_account_cards`  we have 
```
 "metadata": {
      "type": [
        "object",
        "null"
      ]
},
```